test/common/index.js

```diff
@@ -112,6 +112,8 @@ const isOpenBSD = process.platform === 'openbsd';
 const isLinux = process.platform === 'linux';
 const isOSX = process.platform === 'darwin';
 
+const isDumbTerminal = process.env.TERM === 'dumb';
```

Comment:
Should we really expose a property for a one liner? I would just inline it in the tests.